docs/pages/guides/spend-permissions/quick-start.mdx

```diff
@@ -122,57 +56,69 @@ This one's easy if you already have your CDP API key:
 
 ### Create a spender client
 
-Our client is what our app will use to communicate with the blockchain. In this example, our client is a Coinbase Smart Wallet, 
-and we'll use a [paymaster](/guides/paymasters.mdx) to sponsor our transactions so we don't have to worry
-about having ETH in the spender account.
+Our client is what our app will use to communicate with the blockchain.
 
 Create a sibling directory to `app` called `lib` and add the following `spender.ts` file to create your spender client.
 
 ```ts [lib/spender. ts]
-import { createPublicClient, Hex, http } from "viem";
+import { createPublicClient, createWalletClient, Hex, http } from "viem";
 import { baseSepolia } from "viem/chains";
 import { privateKeyToAccount } from "viem/accounts";
-import {
-  createBundlerClient,
-  createPaymasterClient,
-  toCoinbaseSmartAccount,
-} from "viem/account-abstraction";
 
-export async function getSpenderBundlerClient() {
+export async function getPublicClient() {
   const client = createPublicClient({
     chain: baseSepolia,
     transport: http(),
   });
+  return client;
+}
 
-  const spenderAccountOwner = privateKeyToAccount(
-    process.env.SUBSCRIPTION_PRIVATE_KEY! as Hex
+export async function getSpenderWalletClient() {
+  const spenderAccount = privateKeyToAccount(
+    process.env.SPENDER_PRIVATE_KEY! as Hex
   );
 
-  const spenderAccount = await toCoinbaseSmartAccount({
-    client,
-    owners: [spenderAccountOwner],
-  });
-
-  const paymasterClient = createPaymasterClient({
-    transport: http(process.env.BASE_SEPOLIA_PAYMASTER_URL),
-  });
-
-  const spenderBundlerClient = createBundlerClient({
+  const spenderWallet = await createWalletClient({
     account: spenderAccount,
-    client,
-    paymaster: paymasterClient,
-    transport: http(process.env.BASE_SEPOLIA_PAYMASTER_URL),
+    chain: baseSepolia,
+    transport: http(),
   });
-
-  return spenderBundlerClient;
+  return spenderWallet;
 }
 ```
 
 ### Configure the Smart Wallet URL
 
-In `providers.tsx`, update the value of `keysUrl` to be `"https://wallet.chameleon.systems"`.
+In `app/providers.tsx`, update the value of `keysUrl` to be `"https://keys-beta.coinbase.com/connect"`.
```

Comment:
from another PR, this is not `keys-dev.coinbase.com` so make sure to rebase or change this!